docs/install/Knative-with-GKE.md

```diff
@@ -118,8 +118,8 @@ the recommended configuration for a cluster is:
 1. Create a Kubernetes cluster on GKE with the required specifications:
 
 > Note: If this setup is for development, or a non-Istio networking layer (e.g.
-> [Gloo](./Knative-with-Gloo.md)) will be used, then you can remove the
-> `--addons` line below.
+> [Ambassador](https://getambassador.io/) or [Gloo](./Knative-with-Gloo.md)) 
```

Comment:
I think this link to Ambassador should point to the page in Knative Docs, instead of to the Ambassador site.

Reply:
@samodell Good find. Thank you for taking the time to review my work and find that error.